src/routes/createevent.js

```diff
@@ -0,0 +1,20 @@
+const createEvent = require('../handlers/createevent')
+const validateEvent = require('../models/validateEvent.js')
+
+module.exports = (req, res) => {
+  validateEvent(req)
+    .then(() => createEvent(req.body))
+    .then(event => {
+      if (!event) {
```

Comment:
would event ever be falsey? *I cant find the new sql query in this PR to check.* Even if we make a query and nothing is found, normally an empty array is returned, which is truthy.

Reply:
This is to check the query has returned properly.